Makefile

```diff
@@ -10,10 +10,16 @@ compile-windows:
 	go build -o windows/rubixgoplatform.exe
 
 compile-mac:
-	echo "Compiling for MacOS"
+	echo "Compiling for MacOS arm64"
 	go env -w GOOS=darwin
 	go env -w GOARCH=arm64
 	go env -w CGO_ENABLED=1
-	go build -o mac/rubixgoplatform
+	go build -o mac/rubixgoplatform_arm64
+	go env -w GOARCH=amd64
+	go env -w CGO_ENABLED=1
+	go build -o mac/rubixgoplatform_amd64
+
+clean:
+	rm -f linux/rubixgoplatform windows/rubixgoplatform.exe mac/rubixgoplatform_arm64 mac/rubixgoplatform_amd64
```

Comment:
deleting the entire folder will be better, right? That will go a long way when we build or download more files to the build folder.

rm -f linux windows mac

Reply:
done